### DISCUSSION TITLE
Unable to login to crates.io with GitHub

### DISCUSSION THREAD

**Discussion by dormant-user** [rust-lang/crates.io#8691] [Q&A] [answered]
I recently converted my GitHub account from a user profile to an organization, eventually I had to create a new account to manage the organization. This change broke my access to crates.io since it sees the new login as a different account even after verifying the old email address.
Did someone run into a similar issue? What's the fix for it and how do I get access to my account?

**Comment by carols10cents** [comment, marked as answer]
Hi, please email help@crates.io from your verified email addresses and we can get this sorted out!

**Reply by dormant-user** [reply]
@carols10cents Thank you for the response. I'll follow up via email.
